app/logic/events.py

```diff
@@ -2,6 +2,7 @@
 from peewee import DoesNotExist, fn, JOIN
 from dateutil import parser
 from datetime import timedelta, date
+from dateutil.relativedelta import relativedelta
 import datetime
```

Comment:
I know this isn't all you guys but since you use this it would potentially be good if removed this import and added `datetime` to the `from datetime` import statement. That way we could make some of the `datetime.datetime` calls in your function and other places be cleaner and we could remove this import.

Reply:
I saw this and thought about it. Thank you for bringing it up so I can fix it!